### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by pilievOoyala**
This is pulling in the latest version of all the files from our core repo and getting unit tests to work.  The next step will be to rip out these files from the other repo.

### CODE REVIEW

**Comment by aeng7** [comment]
LGTM

File: js/classes/message_bus.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
   (function(OO,_) {
 	/**
```
Comment (angieisrani) [inline js/classes/message_bus.js]: These look to be a merge of changes from the internal repo

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "test": "make test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ooyala/html5-common"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "expect.js": "0.1.2",
+    "hazmat": "*",
```
Comment (angieisrani) [inline package.json]: might be a good idea to lock this at the current version

File: test/unit-test-helpers/message_bus_helper.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+(function(OO,_) {
+
+  _.extend(OO.MessageBus.prototype,  {
+    published: function(event) {
```
Comment (angieisrani) [inline test/unit-test-helpers/message_bus_helper.js]: This looks to be a little behind the internal master-v4 message_bus_helper file
I can send you the link offline

**Comment by angieisrani** [comment]
LGTM!  I think this doesn't contain code coverage but we can get there

**Pull request merged**